src/views/layouts/main.php

```diff
@@ -22,6 +22,20 @@
     <?php $this->registerCsrfMetaTags() ?>
     <title><?= Html::encode($this->title) ?></title>
     <?php $this->head() ?>
+    <style>
+        ul.navbar-nav li, ul.breadcrumb li {
```

Comment:
How about putting that into an asset bundle? `src/assets` with its CSS files also located there.

Reply:
Definitely. I just realized that there is already an outdated site.css file shipped. Should I update it with these (and maybe a little more ui-:heart:)?

Reply:
Yes, would be great to update it.

Reply:
It looks like this now:

![screen shot 2018-12-17 at 11 56 48 pm](https://user-images.githubusercontent.com/304450/50121142-18534a00-0258-11e9-8da8-f5166640d138.png)